packages/terraform-modules-public/aws/testnet/modules/startup-scripts/final-hardening.sh

```diff
@@ -0,0 +1,60 @@
+# Enable autologout
```

Comment:
Nice. Probably we should consider importing this script to the rest of cloud setups.

Reply:
:+1: makes sense.